src/http_controller/inotification_long_polling_http_controller.ts

```diff
@@ -0,0 +1,162 @@
+import {HttpRequestWithIdentity} from '@atlas-engine/http_contracts';
+
+import {Response} from 'express';
+
+/**
+ * Describes a HTTPController for managing HttpRequests related to Notifications.
+ */
+export interface INotificationLongPollingController {
+
+  /*
+  * Waits for the ProcessStarted notification and returns the result
```

Comment:
Würd die Beschreibungen eher so formulieren:

```suggestion
  * Waits for a ProcessStarted notification and resolves with the notification's content.
```

Gilt für alle Header.